core/contracts/common/implementation/PermissionedExpandedERC20.sol

```diff
@@ -0,0 +1,43 @@
+pragma solidity ^0.5.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "./MultiRole.sol";
+import "../interfaces/ExpandedIERC20.sol";
+
+/**
+ * @title An ERC20 with permissioned burning and minting. The contract deployer will initially 
+ * be the only minter and the owner who is capable of adding new roles. 
+ */
+contract PermissionedExpandedERC20 is ExpandedIERC20, ERC20, MultiRole {
```

Comment:
Is there ever going to be a case where we want `ExpandedERC20` and not `PermissionedExpandedERC20`? If so, this is a good case for keeping them seperate. else join them.

Reply:
Yeah @chrismaree I was leaving flexibility in case we wanted a mintable and burnable token but without permissions, but we probably never will to Matt's point about our default expectation

Reply:
@mrice32 changed name